packages/access-key-management-widget/src/lib/widget/api/sdk/index.ts

```diff
@@ -6,8 +6,16 @@ export const createSdk = (
   config: Parameters<typeof createWebSdk>[0],
   tenant: string,
   mock: boolean,
+  widgetId: string,
 ) => {
-  const webSdk = createWebSdk({ ...config, persistTokens: true });
+  const webSdk = createWebSdk({
+    ...config,
+    persistTokens: true,
+    baseHeaders: {
+      'x-descope-widget-type': 'access-key-management-widget',
```

Comment:
Do we need the widget version as well?

Reply:
no, it will be propagat from outside

Reply:
![image](https://github.com/descope/descope-js/assets/5429033/5542a73b-868e-4dae-8ace-5311ec82c7e8)

Reply:
This is the web-sdk version, not the widget version